fe/fe-core/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java

```diff
@@ -1139,4 +1141,21 @@ public Map<Long, Set<Long>> getTableRunningJobMap() {
         return tableRunningJobMap;
     }
 
+    public void processCreateMultiTablesMaterializedView(CreateMultiTableMaterializedViewStmt addMVClause)
+            throws UserException {
+        Map<String, OlapTable> olapTables = addMVClause.getOlapTables();
+        try {
+            olapTables.values().forEach(Table::writeLock);
+            for (OlapTable table : olapTables.values()) {
+                table.checkStableAndNormal(addMVClause.getDatabase().getClusterName());
+                if (table.existTempPartitions()) {
+                    throw new DdlException("Can not alter table when there are temp partitions in table");
```

Comment:
The source table is not modified when creating mtmv, so the check should removed